yocto/recipes-conectivity/ola/ola_git.bb

```
@@ -0,0 +1,73 @@
+SUMMARY = "Open Lighting Architecture - OLA"
+DESCRIPTION = "The Open Lighting Architecture (OLA) is part of the Open Lighting Project and provides applications with a mechanism to send and receive DMX512 & RDM commands using hardware devices and DMX over IP protocols. This enables software lighting controllers to communicate with hardware either via Ethernet or traditional DMX512 networks. \
+OLA can also convert DMX512 data sent using DMX over IP protocols from one format to another, allowing devices from different manufacturers to interact with one another. For example a Strand Lighting Console using ShowNet can send DMX512 to an Enttec EtherGate. When combined with a physical DMX interface such as the DMX USB Pro, OLA can send and receive data from wired DMX512 networks."
+
+LICENSE = "GPLv2 & LGPLv2.1"
+LIC_FILES_CHKSUM = "file://LICENCE;md5=7aa5f01584d845ad733abfa9f5cad2a1"
+
+#DEPENDS_class-nativesdk = "bison libusb1 protobuf protobuf-native pkgconfig-native ossp-uuid ola-native "
+DEPENDS_class-target    = "bison libmicrohttpd avahi libusb1 libftdi cppunit protobuf protobuf-native ola-native"
+DEPENDS_class-native    = "protobuf bison-native flex-native pkgconfig-native ossp-uuid-native"
+
+PV = "0.10.6"
+
+# 0.10.6 tag won't build, this rev has protobuf patches
+SRCREV = "00dc86a48ec4c528cec90166435b440f283a9c86" 
+SRC_URI = "git://github.com/OpenLightingProject/ola.git;protocol=https"
```

We should be able to build 0.10.9 cleanly now, is this the right sort of syntax?
```suggestion
PV = "0.10.9"

SRC_URI = "git://github.com/OpenLightingProject/ola.git;protocol=https"
```